### PR TITLE
JP-2376: GLS one group ramp

### PR DESCRIPTION
**Pull request by kmacdonald-stsci**
The GLS algorithm assumed the group dimension would always be at least two.  This PR handles the special case where the group dimension is one.

Resolves [JP-2376](https://jira.stsci.edu/browse/JP-2376)

### CODE REVIEW

File: src/stcal/ramp_fitting/utils.py [hunk 1]
```diff
@@ -1379,3 +1379,96 @@ def dq_compress_sect(ramp_data, num_int, gdq_sect, pixeldq_sect):
...
+def compute_median_rates(ramp_data):
+    """
+    Compute the median rates for each pixel.
+
+    Parameter
+    ---------
+    ramp_data : RampData
+        Contains the data needed to compute the median rates.
+
+    median_rates : ndarray
```
[the remark this answers is not in the document]
Comment (kmacdonald-stsci) [reply src/stcal/ramp_fitting/utils.py:1393]: I added the Return header to the docstring.

**Review by dmggh — APPROVED** [review]
Looks good to me.

**Review by hbushouse — APPROVED** [review]

**Comment by hbushouse** [comment]
Needs a change log entry.

File: CHANGES.rst [hunk 2]
```diff
@@ -9,6 +9,8 @@ ramp_fitting
 
 - Updating the one good group ramp suppression handler works. [#92]
 
+- Adding special case handler for GLS to handle one group ramps. [#97]
```
Comment (hbushouse) [inline CHANGES.rst:12]: I believe this should go into the "unreleased" section of the file. 0.7.3 is already out.

**Pull request merged**